ngraph/frontend/paddlepaddle/src/op/expand_v2.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "expand_v2.hpp"
+#include <ngraph/opsets/opset6.hpp>
+#include <paddlepaddle_frontend/utility.hpp>
+
+namespace ngraph
+{
+    namespace frontend
+    {
+        namespace pdpd
+        {
+            namespace op
+            {
+                NamedOutputs expand_v2(const NodeContext& node)
```

Comment:
From the doc string [here](https://github.com/PaddlePaddle/Paddle/blob/758dd7bb8f560e45bc49855e4776a9d64c95aea9/paddle/fluid/operators/expand_v2_op.cc#L138). Expand could transform tensor of shape  [2, 3, 1] to the tensor of shape [2, 6, 1]. Dimension 3 is not broadcast-able to 6, please see the [broadcast_rules](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/broadcast_rules.md). According to what I have seen in the doc -- the operation could be expressed through openvino Tile operation -- 
https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/movement/Tile_1.md

Reply:
@luo-cheng2021  Please have a fix

Reply:
There is a conflict between [doc string](https://github.com/PaddlePaddle/Paddle/blob/758dd7bb8f560e45bc49855e4776a9d64c95aea9/paddle/fluid/operators/expand_v2_op.cc#L138) and the ['expand' document](https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/expand_en.html) which says 'The dimension to expand must have a value 1.'
Anyway, I'll use the 'Tile' op to support all cases.